web/template/create-draft.gohtml

```diff
@@ -0,0 +1,227 @@
+{{ template "page" . }}
+
+{{ define "title" }}Create a draft LPA{{ end }}
+
+{{ define "main" }}
+  {{ if .Success }}
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title">
+        {{ if (eq (len .Uids) 2) }}
+          Draft Health and Welfare and Property and Finance LPAs for the donor
+          {{ .Draft.DonorFirstname }} {{ .Draft.DonorMiddlename }} {{ .Draft.DonorSurname }} have been saved
```

Comment:
Would this leave an extra space if the donor did not have a middlename?

Reply:
Yes, but brwosers automatically compress the whitespace down into 1 space, so you can do stuff like:
```html
<p>
  Here is a really long paragraph, that
  I want to write over multiple lines with
  indentation. But I don't want the
  whitespace in the output!
</p>
```